css/properties/display.json

```diff
@@ -142,9 +142,15 @@
               "safari_ios": {
                 "version_added": "11.3"
               },
-              "samsunginternet_android": {
-                "version_added": true
-              },
+              "samsunginternet_android": [
+                {
+                  "version_added": "9.0"
+                },
+                {
+                  "version_added": "7.0",
+                  "version_removed": "9.0"
```

Comment:
This probably needs a note for what was changed. 

Between version 7.0 and 9.0